verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -127,7 +128,7 @@ export default class SourcesController {
         // Careful here, source.isModified('automation.schedule.awsScheduleExpression')
         // will return true even when just the parser is updated which is
         // error prone, prefer isModified() without dotted.paths if possible.
-        if (source.automation?.schedule?.isModified('awsScheduleExpression')) {
+        if (source.automation?.isModified('schedule')) {
```

Comment:
This had to be changed. This is identical in practice (the only other schedule field is rule ARN -- which won't change) in the case of adding/changing automation. But, if automation is removed -- meaning the schedule is removed (since awsScheduleExpression is required) -- we'll never trigger this scenario since schedule? is falsey.

Reply:
ack, thanks for the explanation